src/conversions.jl

```diff
@@ -479,6 +479,55 @@ function convert_arguments(
     (m,)
 end
 
+
+
+function convert_arguments(P::PlotFunc, r::AbstractVector, f::Function)
+    ptype = plottype(P, Lines)
+    to_plotspec(ptype, convert_arguments(ptype, r, f.(r)))
+end
+
+function convert_arguments(P::PlotFunc, i::AbstractInterval, f::Function)
+    x, y = PlotUtils.adapted_grid(f, endpoints(i))
+    ptype = plottype(P, Lines)
+    to_plotspec(ptype, convert_arguments(ptype, x, y))
+end
+
+to_tuple(t::Tuple) = t
+to_tuple(t) = (t,)
+
+function convert_arguments(P::PlotFunc, f::Function, args...; kwargs...)
+    tmp = f(args...; kwargs...) |> to_tuple
```

Comment:
to_tuple should already exist somewhere, and please dont use `|>`

Reply:
That's the only place it's defined, when I removed it and forgot to paste back I got an undefvar error for it.
```suggestion 
    tmp =to_tuple(f(args...; kwargs...))

Reply:
We could relocate it somewhere else?

Reply:
Ah lol, you just moved it around - i guess I meant exactly that definition then :D well, then lets just remove the pipe :D

Reply:
done :)